jormungandr-integration-tests/src/common/jcli_wrapper/jcli_transaction_wrapper/jcli_transaction_commands.rs

```diff
@@ -157,4 +181,14 @@ impl TransactionCommands {
             .arg(staging_file.as_os_str());
         command
     }
+
+    pub fn get_transaction_info_command(&self, staging_file: &PathBuf) -> Command {
+        let mut command = Command::new(configuration::get_jcli_app().as_os_str());
+        command
+            .arg("transaction")
+            .arg("info")
+            .arg("--staging")
+            .arg(staging_file.as_os_str());
+        command
```

Comment:
you should be able to provide a custom format for the output you want to extract (if that makes your life easier).